biopet-core/src/main/resources/nl/lumc/sasc/biopet/utils/rscript/plotScatter.R

```diff
@@ -1,40 +0,0 @@
-library(reshape2)
```

Comment:
Is the functionality of these R scripts taken over by something else? If not, that leaves us with fewer plots.